.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,82 @@
+name: Build Docker
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - 'v*'
+
+
+jobs:
+  build_dev:
+    name: Deploy dev
+    if: startsWith(github.ref, 'refs/heads/') # Just the branches
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set environment
+        run: |
+          export TAG="${GITHUB_REF#'refs/tags/'}"
+          echo "::set-env name=GH_TAG::$TAG"
+          echo "::set-env name=DOCKER_REPO::${{ secrets.DOCKER_USERNAME }}/dungeon-revealer"
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          buildx-version: latest
+          qemu-version: latest
+
+      - name: Available platforms
+        run: echo ${{ steps.buildx.outputs.platforms }}
+
+      - name: Login to DockerHub Registry
+        run: echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+
+      - name: Run Buildx
+        run: |
+          docker buildx build \
+            --platform linux/amd64,linux/386,linux/arm/v6,linux/arm/v7,linux/arm64 \
```

Comment:
Could we reduce the duration by using a matrix, similar to the node.js version matrix?

Reply:
I'll give it a try.

Reply:
I managed to reduce the duration to ~1 hour! I had to do some caching between the jobs to achieve it.